### PR TITLE
Fixed prefix and input misalignment and line break in prefix

### LINKED ISSUE #1724: Fix misalignment between prefix and input on large screens. prefix UI breaking when hyphens are added in text.
1. image
2. image

### PR DESCRIPTION
**Pull request by Calvinwilson99**
- Fixes #1724 

**Description**

- Fixed: Misalignment between input and prefix for large screens. Line break in prefix when hyphens are added.

**Checklist**

- ~~[ ] I have made corresponding changes to the documentation.~~
- ~~[ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

### CODE REVIEW

**Comment by Calvinwilson99** [comment]
@praveen-murali-ind _a. Please review. 
I'm not exactly sure how to test the alignment issue, since it occurs only on large screens. I tried setting the screen resolution to a higher value but couldn't see the misalignment.
I have tested that that prefix and suffix does not break to a new line on giving text with a hyphen.

File: src/styles/components/_input.scss [hunk 1]
```diff
@@ -65,10 +65,6 @@
       }
     }
 
-    input {
-      line-height: 1.2;
```
Comment (praveen-murali-ind) [inline src/styles/components/_input.scss:69]: @Calvinwilson99 _a Please revert this change.

**Review by praveen-murali-ind — APPROVED** [review]
@Calvinwilson99 _A LGTM. A minor change. Please revert the line height changes in input CSS.

**Comment by Calvinwilson99** [comment]
@amaldinesh7 _a. Please review.
